manifests/redis_exporter.pp

```diff
@@ -82,15 +90,17 @@
 #  The binary release version
 
 class prometheus::redis_exporter (
-  Array[String] $addr,
   String $download_extension,
   String $download_url_base,
-  Array[String] $extra_groups,
   String $group,
+  Array[String] $extra_groups,
   String $package_ensure,
   String $package_name,
   String $user,
   String $version,
+  String $redis_exporter_config_file,
+  Array[String] $addr,
```

Comment:
is  there a reason for reordering the existing parameters?

Reply:
Hi @bastelfreak , the only reason is to make things easy to read grouping variables by its type.